setup.cfg

```diff
@@ -33,43 +33,60 @@ include_package_data = True
 setup_requires =
   setuptools_scm
 install_requires =
-  numpy>=1.16.0
-  scipy>=1.2.0
-  matplotlib>=2.2.2
-  pandas>=0.23.0
   astropy>=4.0.3
+  numpy>=1.16.0
   parfive[ftp]>=1.1.0
-  h5netcdf>=0.8.1
 
 [options.extras_require]
-database = sqlalchemy
+asdf =
+  asdf
+dask =
+  dask[array]
+database =
+  sqlalchemy
+instr =
+  matplotlib>=2.2.2
+  pandas>=0.23.0
+  scipy>=1.2.0
 image =
   scikit-image
-jpeg2000 = glymur!=0.9.0
+  scipy>=1.2.0
+jpeg2000 =
+  glymur!=0.9.0
+map =
+  matplotlib>=2.2.2
+  scipy>=1.2.0
 net =
   beautifulsoup4
   drms
+  pandas>=0.23.0
   python-dateutil
-  zeep
   tqdm
-asdf = asdf
-dask = dask[array]
+  zeep
+time =
+  pandas>=0.23.0
```

Comment:
I don't think we need this, you either have pandas installed and want to parse pandas timestamps or you don't. You aren't ever going to want to install pandas just for `parse_time`?
```suggestion
```

Reply:
Fixed.